src/plugins/toolbar/prism-toolbar.ts

```diff
@@ -1,6 +1,6 @@
 import { getParentPre } from '../../shared/dom-util';
 import { noop } from '../../shared/util';
-import type { CompleteEnv, HookCallback } from '../../core/hooks';
+import type { CompleteEnv, HookCallback } from '../../core/classes/hooks';
```

Comment:
Some new type not yet defined? I mean `CompleteEnv`.

Reply:
No idea.